pages/faq.vue

```diff
@@ -245,7 +245,7 @@ export default {
             question: "How does the Covid Watch app protect my privacy?",
             answer:
               "<p>In short, we don’t collect data about who you are, where you go or who you’re near. With the Covid Watch app, your phone periodically sends out Bluetooth signals, essentially saying to nearby phones, “Hey, I’m here!” If another phone nearby also has the app and says, “Hey, I’m here too,” both phones will log the interaction and store it for two to four weeks right there inside your device.</p>" +
-              "<p>If you are diagnosed with COVID-19, you can choose to share your test results with other Covid Watch users after they are verified by local health officials.  Once you do that, you upload a random number to our database that allows other devices to recognize that the Bluetooth numbers they list are associated with a positive result. The database contains no personal information and no locations. A hacker who broke into our database or your phone would just find a list of random numbers.</p>"
+              "<p>If you are diagnosed with COVID-19, you can choose to share your test results with other Covid Watch users. Once your results have been verified by local health officials, your phone will upload a random number to our database. Other Covid Watch users whose phones have stored the same random number will then be notified that they were exposed to someone who tested positive for COVID-19. Our database contains no personal information and no locations. A hacker who broke into our database or your phone would just find a list of random numbers.</p>"
```

Comment:
This sentence went through a few different iterations in previous PRs (https://github.com/covid19risk/website/pull/66, https://github.com/covid19risk/website/pull/67, https://github.com/covid19risk/website/pull/68).

I like the first part of this rewording, which takes us from:
> If you are diagnosed with COVID-19, you can choose to share your test results with other Covid Watch users after they are verified by local health officials.  Once you do that, you upload a random number to our database 

to

> If you are diagnosed with COVID-19, you can choose to share your test results with other Covid Watch users. Once your results have been verified by local health officials, your phone will upload a random number to our database.

The second part isn't strictly true, though- this is getting a bit into the weeds of [the TCN Protocol](https://github.com/TCNCoalition/TCN#reporting), but for scalability reasons, the number that someone shares with the database will not actually be the same number that phones have stored. I agree that the current wording is confusing, though. Maybe

> you upload a random number to our database that allows other devices to recognize that the Bluetooth numbers they list are associated with a positive result

could become

> your phone will upload a random number to our database. Other devices will be able to download this number and see whether they received a Bluetooth chirp from your phone while you might have been infectious. If so, an exposure notification will be displayed.

Reply:
It seems "device" is used interchangeably with "phone". Can the app be installed on anything other than a phone? If not it might be best to stick with "phone" so as not to confuse users. I would also suggest replacing "and" with "to":

`Other devices will be able to download this number to see whether they received a Bluetooth chirp from your phone while you might have been infectious.`